src/encryptsave/mod.rs

```diff
@@ -0,0 +1,175 @@
+mod ll;
+pub mod errors;
+
+
+macro_rules! tox_res {
+    ( $res:ident <- $rexpr:expr, $err:ident, $r:expr ) => {
```

Comment:
`$res` and `$err` in fact is the same, well,...
It looks like this? https://gist.github.com/quininer/1620ab00aeeb82f30e88